config/config.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import (
+	"os"
+	"strconv"
+)
+
+// Config holds the application configuration
+type Config struct {
+	RedisAddr     string
+	ServerPort    string
+	RequestLimit  int // Field for the request limit
+	RequestWindow int // Field for the time window in seconds
+}
+
+// LoadConfig loads the application configuration from environment variables or defaults
+func LoadConfig() *Config {
+	return &Config{
+		RedisAddr:     getEnv("REDIS_ADDR", "localhost:7379"), // Default Redis address
```

Comment:
This is not required, we'd be using DiceDB.

Reply:
Made the Changes